lib/galaxy/model/__init__.py

```diff
@@ -1370,7 +1371,7 @@ class Job(Base, JobLike, UsesCreateAndUpdateTime, Dictifiable, Serializable):
 
     states = JobState
 
-    terminal_states = [states.OK, states.ERROR, states.DELETED]
+    terminal_states = [states.OK, states.ERROR, states.DELETED, states.DELETING]
```

Comment:
I find confusing to include `DELETING` here, since I'd expect that the job state changes to DELETED at a certain point, so DELETING is not terminal.

Reply:
I can rename it to almost_terminal_states 😅 I'll see what the alternatives are

Reply:
Could be added explicitly when you are checking for (almost) terminal states?

Reply:
Moved that the finished_states, which we already use in the `finished` property

Reply:
Perfect!